.github/workflows/tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         # Not all Python versions are avalaible for linux AND x64
         # https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9, '3.10']
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
Is Python 3.4 supposed to be a part of this? It is included in the areas below.

Reply:
I thought when @redtoad added the CI configuration in #6, he mentioned that Github Actions doesn't offer environments for Python 3.3 and 3.4, that's why they're not included here. (I can't find the comment now, though, so maybe I'm remembering that wrong?)